Vagrantfile

```diff
@@ -0,0 +1,26 @@
+Vagrant.configure("2") do |config| 
+    config.vm.box = "hashicorp/bionic64"
+    config.vm.network "forwarded_port", guest: 5000, host: 5000
+    config.vm.provision "shell", privileged: true, inline: <<-SHELL 
+    config.vm.provision "shell", path: "scripts/setup_ssh.sh"
```

Comment:
I think this second provisioning line (line 5) is a copy-paste error, it just quietly fails currently but it would be nice to remove that

Reply:
removed this line